packages/gen-ai-hub/jest.config.mjs

```diff
@@ -1,5 +1,7 @@
 import config from '../../jest.config.mjs';
-export default {
+const genAiConfig = {
   ...config,
   displayName: 'gen-ai-hub',
 };
+
+export default genAiConfig;
```

Comment:
[q/pp] Why? Really really minor, but because this file will potentially be copy pasted to new packages, I prefer the direct export.